src/styles/blocks/header.scss

```diff
@@ -0,0 +1,46 @@
+.header {
```

Comment:
Consider specifying the size of the link 
<img width="177" alt="Screenshot 2024-04-18 at 16 56 35" src="https://github.com/mate-academy/layout_catalog/assets/135232001/62535153-5426-4c45-956f-c062a1e4707c">